### ISSUE TITLE
Update dependency version for @expo/config-plugins

### ISSUE THREAD

**Issue by lawnguyen** [revtel/react-native-nfc-manager#695]
Need to update to support @expo/config-plugins:"~7.8.0".

**Comment by RonakDoshiTMI** [comment]
+1

<img width="722" alt="Screenshot 2024-01-26 at 2 42 49 PM" src="https://github.com/revtel/react-native-nfc-manager/assets/141172434/d36369e8-5cdb-4e4d-906a-82d16c77d618">

**Comment by hezzze** [comment]
+1

![image](https://github.com/revtel/react-native-nfc-manager/assets/2413682/cefca963-f4f5-421a-bd61-07240a40e5f5)

**Comment by nfuenmamelt** [comment]
any solutions for this

**Comment by mwiedemeyer** [comment]
Can someone please update the @expo/config-plugins dependency to 8.0.0? Its required for Expo SDK51
![image](https://github.com/revtel/react-native-nfc-manager/assets/4295189/56263dab-8ad5-46f4-81ba-f44e3b1f7cef)

See: https://github.com/revtel/react-native-nfc-manager/blob/9301ab3f8d13115614d11ea6f1f9da22697835f1/package.json#L65

**Comment by fredlemieux** [comment]
Please see PR #732, tests pass, please let me know if there is anything else I need to do?  Do I need to bump the version number? I looks like this is done automatically using "release-it"

**Comment by fredlemieux** [comment]
There are loads of outdated packages and vulnerabilities, I am happy to raise another PR fixing these once this is merged, which is a priority...

**Comment by fredlemieux** [comment]
> There are loads of outdated packages and vulnerabilities, I am happy to raise another PR fixing these once this is merged, which is a priority...

#733 <- Here it is...
